az-hadoop-jobtype-plugin/src/main/java/azkaban/jobtype/HadoopTuningSecurePigWrapper.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright 2018 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.jobtype;
+
+import static org.apache.hadoop.security.UserGroupInformation.HADOOP_TOKEN_FILE_LOCATION;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.security.PrivilegedExceptionAction;
+import java.util.Properties;
+
+import org.apache.hadoop.security.UserGroupInformation;
+import org.apache.log4j.Logger;
+import org.apache.pig.tools.pigstats.PigStats;
+
+import azkaban.jobtype.pig.PigUtil;
+import azkaban.jobtype.tuning.TuningCommonConstants;
+import azkaban.jobtype.tuning.TuningErrorDetector;
+import azkaban.jobtype.tuning.TuningException;
+import azkaban.jobtype.tuning.TuningParameterUtils;
+import azkaban.utils.Props;
+
+
+/**
+ * This class represent wrapper for running pig job with tuning enabled.
+ */
+public class HadoopTuningSecurePigWrapper {
+
+  public static final String WORKING_DIR = "working.dir";
```

Comment:
I wonder if there is a global working dir config defined somewhere else that can be used here instead of a custom one just for HadoopTuningSecurePigWrapper?